components/insight/SRC/org/openmicroscopy/shoola/env/data/AdminServiceImpl.java

```diff
@@ -872,4 +877,14 @@ public boolean isSystemUser(long userID, String key)
         }
         throw new IllegalArgumentException("Key not valid.");
     }
+
+    /**
+     * Implemented as specified by {@link AdminService}.
+     * @see AdminService#lookupLdapAuthExperimenter(SecurityContext, long)
+     */
+    public String lookupLdapAuthExperimenter(SecurityContext ctx, long userID)
+            throws DSOutOfServiceException, DSAccessException
+    {
+        return gateway.lookupLdapAuthExperimenter(ctx, userID);
```

Comment:
Ah, understood. Storing it in the DB would get us back into the de-synchronization trouble we had previously. So barring a larger reworking (like **all** users are in LDAP), this will have to stay.